lib/modules/manager/glasskube/index.ts

```diff
@@ -0,0 +1,10 @@
+import type { Category } from '../../../constants';
+import { GlasskubePackagesDatasource } from '../../datasource/glasskube-packages';
+import * as GlasskubeVersioning from '../../versioning/glasskube';
+
+export { extractAllPackageFiles, extractPackageFile } from './extract';
+export const defaultConfig = {
+  versioning: GlasskubeVersioning.id,
```

Comment:
```suggestion
  fileMatch: [],
```

versioning is set by datasource, so not needed here